md/work-execution-audit.md

```diff
@@ -0,0 +1,61 @@
+# Work execution audit
+
+Sometimes it is useful to understand what is currently happening in the platform, in particular for work execution, see [Work service mechanism](execution-sequence-states-and-transactions.md).
+
+This can be achieved using a specific logger called _Work execution audit_
+
+
+## How to activate it
+
+### Enable the logger
+
+Enable the logger called `BONITA_WORK_AUDIT.EXECUTION` in the logger configuration.
+
+
+### Activate and configure the logger
+
+In `bonita-tenant-community-custom.properties` set flag `bonita.tenant.work.audit.activated` to `true`
+
+
+Some properties can be configured
+
+* `bonita.tenant.work.audit.abnormal.execution.threshold.execution_count`: will trigger a warning if this amount of reschedule is reached
+* `bonita.tenant.work.audit.abnormal.execution.threshold.execution_count_duration_amount`: will not trigger the warning age of work is less than this value
+* `bonita.tenant.work.audit.abnormal.execution.threshold.execution_count_duration_unit`: unit of time in ChronoUnit (SECONDS, MINUTES, HOURS, ...)
+* `bonita.tenant.work.audit.abnormal.execution.threshold.elapsed_duration_since_registration_amount`: will trigger a warning ifif the work age is greater than this amount
+* `bonita.tenant.work.audit.abnormal.execution.threshold.elapsed_duration_since_registration_unit`: unit of time in ChronoUnit (SECONDS, MINUTES, HOURS, ...)
+
+Example configuration:
+
+```properties
+# Work execution audit
+# Durations must be ChronoUnit enum values
+bonita.tenant.work.audit.activated=true
+# Too much executions threshold
+bonita.tenant.work.audit.abnormal.execution.threshold.execution_count=10
+bonita.tenant.work.audit.abnormal.execution.threshold.execution_count_duration_amount=10
+bonita.tenant.work.audit.abnormal.execution.threshold.execution_count_duration_unit=MINUTES
+# Too much time elapsed since registration
+bonita.tenant.work.audit.abnormal.execution.threshold.elapsed_duration_since_registration_unit=MINUTES
+bonita.tenant.work.audit.abnormal.execution.threshold.elapsed_duration_since_registration_amount=30
+```
+
+
+## What can be tracked
```

Comment:
ℹ️ I would put this at the begining of the page to 1st talk about the what before the how

Reply:
I agree with @tbouffard.

Reply:
done